cmd/gorepomod/internal/arguments/args.go

```diff
@@ -36,7 +36,6 @@ var (
 		"docs",
 		"examples",
 		"hack",
-		"plugin",
```

Comment:
Jeff specifically added this in https://github.com/kubernetes-sigs/kustomize/pull/3910. Why? 😕